src/lib/components/editor/EditorTop.tsx

```
@@ -0,0 +1,105 @@
+          px={2}
+          pb={1}
+          pt={4}
+          borderTop="1px"
+          borderTopColor="gray.700"
+        >
+          {selectedFile.path.split("/").map((path, index) => (
+            <>
+              {index !== 0 && (
+                <CustomIcon name="chevron-right" boxSize={3} color="gray.600" />
+              )}
+              <Text color="gray.600" variant="body3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)